test/handler/test_flow_handler.vader

```diff
@@ -100,31 +111,124 @@ Before:
   \ "passed": v:false
   \}
 
-  runtime ale_linters/javascript/flow.vim
-
-After:
-  unlet! g:flow_output
-  unlet! g:expected
-  unlet! g:actual
-  call ale#linter#Reset()
-
-Execute(The flow handler should process errors correctly.):
-  let g:actual = ale_linters#javascript#flow#Handle(347, [json_encode(g:flow_output)])
+  let g:actual = ale_linters#javascript#flow#Handle(bufnr(''), [json_encode(g:flow_output)])
   let g:expected = [
   \ {
   \   'lnum': 417,
-  \   'bufnr': 347,
   \   'type': 'E',
   \   'col': 10,
   \   'text': 'number: This type is incompatible with the expected return type of array type',
   \ },
   \ {
   \   'lnum': 419,
-  \   'bufnr': 347,
   \   'type': 'W',
   \   'col': 3,
   \   'text': 'unreachable code:',
   \ },
   \]
 
   AssertEqual g:expected, g:actual
+
+Execute(The flow handler should fetch the correct location for the currently opened file, even when it's not in the first message.):
```

Comment:
Nah, that's fine. It's better to have the Vader lines be long and descriptive, than being hard to understand.